server/models/user.coffee

```diff
@@ -21,6 +21,9 @@ module.exports = User = cozydb.getModel 'User',
     owner: Boolean
     allow_stats: Boolean
     activated: Boolean
+    otp_key: String
```

Comment:
Fields in JS are written with camel cases. The previous fields should not be written with snake case. It's a mistake we did a long time ago.